support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -52,13 +52,29 @@ const pageQaId = 'qa-paper-subscriptions';
 function PaperLandingPage({
 	productPrices,
 	promotionCopy,
-}: PaperLandingContentPropTypes) {
+	participations,
+}: PaperLandingPropTypes) {
 	const sanitisedPromoCopy = getPromotionCopy(promotionCopy);
-	const fulfilment: PaperFulfilmentOptions = window.location.pathname.includes(
-		'delivery',
-	)
-		? HomeDelivery
-		: Collection;
+
+	const isNationalDeliveryAbTestVariant =
+		participations.nationalDelivery === 'variant';
+
+	const [shouldRedirectToHomeDelivery, setShouldRedirectToHomeDelivery] =
+		useState<boolean>(isNationalDeliveryAbTestVariant);
+
+	if (
+		shouldRedirectToHomeDelivery &&
+		!window.location.pathname.includes('delivery')
+	) {
+		window.history.replaceState({}, '', paperSubsUrl(true));
```

Comment:
Mind if we include the argument name, to make it clearer what it means?

```suggestion
		window.history.replaceState({}, '', paperSubsUrl(withDelivery = true));
```

Reply:
alas this isn't a Javascript feature (unless I'm mistaken) but I wish it was

Reply:
Oh I see! Glad one of us knows 🙃

Reply:
Probably "works" because assignment returns the value, a la C? Anyway, no worries